src/cpp/session/resources/schema/user-prefs-schema.json

```diff
@@ -515,6 +515,11 @@
             "title": "Highlight R function calls",
             "description": "Whether to highlight R function calls in the code editor."
         },
+        "rainbow_parentheses": {
+            "type": "boolean",
+            "default": false,
```

Comment:
Add a `title` to this (and regenerate prefs) so it can be set from the Command palette.

Reply:
(note you'll probably have to merge master into your branch to get this capability)